evaluator.fhir/src/main/java/org/opencds/cqf/cql/evaluator/fhir/test/TestRepository.java

```diff
@@ -0,0 +1,27 @@
+package org.opencds.cqf.cql.evaluator.fhir.test;
+
+import java.util.List;
+
+import org.hl7.fhir.instance.model.api.IBaseBundle;
+import org.opencds.cqf.cql.evaluator.fhir.repository.InMemoryFhirRepository;
+
+import ca.uhn.fhir.context.FhirContext;
+
+/**
+ * This class has been deprecated. Use InMemoryFhirRepository instead.
```

Comment:
@barhodes could there be a reason why you did not use the [deprecated annotation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Deprecated.html) for this class?

Reply:
Only my ignorance of that annotation.  I will get it added in.  Thanks!